Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -612,6 +612,42 @@ def create_contact():
     demisto.results(demisto_entry)
 
 
+def create_group():
+    assert conn is not None
+    args = demisto.args()
+
+    object_classes = ["top", "group"]
+    dn = args.get('dn')
+    group_name = args.get('name')
+    group_class = args.get("group-class")
```

Comment:
@PM-IST-PC 
what is the purpose of this argument? it is not being used.

please update the code or remove the argument.